02. 1, 2차원 배열 탐색/03. 가위바위보/김민영/index.js

```diff
@@ -1,7 +1,17 @@
 function solution(a, b) {
   let answer = '';
 
-  return answer;
+  for (let i = 0; i < a.length; i++) {
+    if (a[i] === b[i]) {
+      answer += 'D\n';
+    } else if ((a[i] === 1 && b[i] === 3) || (a[i] === 2 && b[i] === 1) || (a[i] === 3 && b[i] === 2)) {
+      answer += 'A\n';
+    } else {
+      answer += 'B\n'
+    }
```

Comment:
코드 리뷰가 없어서 허전할까봐 굳이 굳이 continue문을 예시를 남겨보았습니다~

```suggestion
    if (a[i] === b[i]) {
      answer += 'D\n';
      continue;
    } 
    
    if ((a[i] === 1 && b[i] === 3) || (a[i] === 2 && b[i] === 1) || (a[i] === 3 && b[i] === 2)) {
      answer += 'A\n';
      continue;
    }
    
    answer += 'B\n'
    
```